### ISSUE TITLE
A11y_TrydotNetWebsite_Screen Reader: Narrator is not reading left/right brackets present in code section

### ISSUE THREAD

**Issue by shraddhaapatil** [dotnet/try#961]
GitHubTags:#A11ySev2;#A11yTCS;#A11yWCAG2.1;#DesktopWeb;#.NET Websites;#Win10;#P2-TrydotNetWebsite-Aug21;#WCAG1.3.1;#Info and Relationships;#FTP:#P2-WCP;#Screen Reader;#Narrator

### Actual Result:
Narrator is not reading left/right brackets present in code section.

### Expected Result: ​
Narrator should read all left/right brackets present in code section.

### Environment Details:​
Application Name: TrydotNetWebsite
Standard Violated: WCAG1.3.1

### Note:

No issues with NVDA.

### Repro Steps:​
1.Hit the URL https://try.dot.net
2.Tab till curly brackets.
3.Verify whether narrator is reading left/right brackets present in code or not.

### User Impact

Screen Reader users will face difficulty in navigation if Narrator is not reading left/right brackets present in code section.

![961_A11y_TrydotNetWebsite_Screen Reader Narrator is not reading leftright brackets present in code section](https://user-images.githubusercontent.com/90324695/133957183-6aa67fce-2248-42f3-865f-826faadd6a67.gif)

![961_No issues with NVDA](https://user-images.githubusercontent.com/90324695/133957358-ba6d8c0e-9597-4066-a6f2-f5ec98345d51.gif)

**Comment by shraddhaapatil** [comment]
![961_A11y_TrydotNetWebsite_Screen Reader Narrator is not reading leftright brackets present in code section](https://user-images.githubusercontent.com/90324695/133416142-caa64d15-0114-4c97-9a12-9a8478b0a02b.gif)

**Comment by shraddhaapatil** [comment]
![961_No issues with NVDA](https://user-images.githubusercontent.com/90324695/133416316-d6890d9c-d406-4998-ace9-500e57272749.gif)

**Comment by fsteffi** [comment]
https://learn.microsoft.com/en-us/dotnet/csharp/tour-of-csharp/tutorials/hello-world?tutorial-step=1&WT.mc_id=-blog-scottha
issue still reproduces

@terrimorton

**Comment by AnKushSingh05** [comment]
#Closed;
Verified the issue in below environment. Issue is fixed hence closing the bug.
https://learn.microsoft.com/en-us/dotnet/csharp/tour-of-csharp/tutorials/hello-world?tutorial-step=1&WT.mc_id=-blog-scottha

https://github.com/user-attachments/assets/5bf76c17-05a4-458c-b3c9-829fc595b911
